release 22.2: roachtest: fix lib/pq blocklist

ZhouXing19 opened the pull request:
Update the block list so that there's only one for each version.

Epic: None
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/97050)
<!-- Reviewable:end -->

ZhouXing19 committed:
release 22.2: roachtest: fix lib/pq blocklist

Update the block list so that there's only one for each version.

Epic: None
Release note: None

ZhouXing19 commented:
Closing it and in favor of #98493
